core/com.b2international.snowowl.datastore/src/com/b2international/snowowl/datastore/request/RepositoryCommitRequestBuilder.java

```diff
@@ -84,18 +85,5 @@ public final RepositoryCommitRequestBuilder setParentContextDescription(String p
 	protected final Request<BranchContext, CommitResult> doBuild() {
 		return new TransactionalRequest(author, commitComment, getBody(), preparationTime, parentContextDescription);
 	}
-	
-	public AsyncRequest<CommitResult> build(String repositoryId, String branch) {
```

Comment:
Is there any replacement for this `build` method?

Reply:
This was the same `build` method as the one in `RevisionIndexRequestBuilder`.